lib/ohai/plugins/windows/virtualization.rb

```diff
@@ -44,6 +44,14 @@
       virtualization[:system] = "kvm"
       virtualization[:role] = "guest"
       virtualization[:systems][:kvm] = "guest"
+    when "American Megatrends Inc."
```

Comment:
I am fairly new to ruby, are you ok with this - if bios[0]["version"].include? "VRTUAL -" ...